openvdb_houdini/houdini/SOP_NodeVDB.cc

```diff
@@ -234,6 +234,22 @@ SOP_NodeVDB::fillInfoTreeNodeSpecific(UT_InfoTree& tree, const OP_NodeInfoTreePa
     if (UT_InfoTree* child = tree.addChildMap("OpenVDB")) {
         child->addProperties("OpenVDB Version", openvdb::getLibraryAbiVersionString());
     }
+
+    UT_StringArray sparseVolumeTreePath({"SOP Info", "Sparse Volumes"});
+    if (UT_InfoTree* sparseVolumes = tree.getDescendentPtr(sparseVolumeTreePath)) {
+        if (UT_InfoTree* info = sparseVolumes->addChildBranch("OpenVDB Points")) {
```

Comment:
The purpose of this branch is to add the openvdb specific stuff that hasn't made it to the Houdini baseline.  So if you want to add some new Foo property, you can add it to the OpenVDB Points branch and adjust your mako template to displ it.  It happens to be rendundant in 17 because we finally imported the changes, but it is needed to access the data in earlier versions.  It is also useful if you need something new, like "number of triangles" that won't yet exist.

Reply:
I don't think we should include redundant features, it would be better to add a Houdini 17 ifdef to disable this going forward. If we need something like "number of triangles", then we should add just that on it's own branch and customize the mako to pick up that one piece of information and get the rest from the native branch. 

I don't really like duplicating any information that's already included natively, because the data then needs to be manually kept in-sync for it to be at all useful. In fact, this is already out-of-date because the names don't match ("Count" != "Point Count"), so this seems just to be creating an extra maintenance burden.

Reply:
The counter argument is that to add the number of triangles, you'll end up rewriting all this code as you need it synced with the other attributes anyways.   I like it being here, rather than in a remote branch, as it will at least make it easy to learn how to build these extra trees.  Having extra branches on the extended type info isn't a hardship as they'll just be ignored by the standard mako code.

But I won't fight strongly to keep it either.